locales/de.json

```diff
@@ -454,27 +454,27 @@
 				"displayName": "Anzeige-Name",
 				"domainName": "Domain-Name",
 				"domainName_pathInLDAP": "Domain-Name (Pfad im LDAP)",
-				"fullPathClassUsers": "Nutzer:in der Klasse mit vollem Pfad",
+				"fullPathClassUsers": "Nutzer:innen der Klasse mit vollem Pfad",
 				"hide": "Ausblenden",
 				"noSchoolCloud": "keine HPI Schul-Cloud",
-				"noSchoolCloudTitle": "Dies ermöglicht es Ihnen, dass Nutzer:in aus Ihrem LDAP nicht in die HPI Schul-Cloud übertragen werden",
+				"noSchoolCloudTitle": "Dies ermöglicht es Ihnen, dass Nutzer:innen aus Ihrem LDAP nicht in die HPI Schul-Cloud übertragen werden",
```

Comment:
dir, deinem

Dies ermöglicht es dir, dass Nutzer:innen aus LDAP nicht in die HPI Schul-Cloud übertragen werden